user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Tables/Table_component/Filtering_Table_Content.md

```diff
@@ -0,0 +1,85 @@
+---
+uid: Filtering_Table_Content
+---
+
+# Filtering table content
+
+From DataMiner 10.2.7/10.3.0 onwards, you can filter the contents of a table component using one of the available three methods: the [general filter](#general-filter), the [column-based filter](#column-based-filter), or a [text string](#filter-based-on-text-string).
+
+## General filter
+
+To apply a general filter across the table, a **search box** is available:
+
+1. Click the search icon in the top-right corner.
+
+1. Specify the filter text (case-insensitive) in the search box.
+
+   ![General filter](~/user-guide/images/General_Filter.gif)<br>*Table component in DataMiner 10.5.6*
+
+   This will apply a client-side filter only. To apply a server-side filter, you need to use a filter operator when you [configure the query data source](xref:Creating_GQI_query).
+
+> [!IMPORTANT]
+> From DataMiner 10.3.0 [CU20]/10.4.0 [CU8]/10.4.11 onwards<!-- RN 40818-->, the search box is only available when the [*Show quick filter* setting](xref:DashboardTable#table-layout) is enabled.
+
+## Column-based filter
+
+To apply **a filter based on a specific column**:
+
+1. Right-click the column header and select *Filter*.
+
+1. Configure the different fields of the filter, depending on the type of value in the column.
+
+   - For string values or GUIDs:
+
+     - To switch between a positive or negative filter, click *does* or *does not*.
+
+     - To switch to a different type of filter, click the second filter field. This will toggle between *contain*, *equal*, and *match regex*.
+
+     - In the third field of the filter, specify a filter value.
+
+   - For numeric or datetime values, specify the range that a value should be in.
+
+   - For booleans, specify whether the value should be true or false.
+
+   - For discrete values, from DataMiner 10.2.10/10.3.0 onwards, select the relevant checkboxes.
+
+   > [!NOTE]
+   > Prior to DataMiner 10.2.9/10.3.0, it is possible to specify multiple conditions by clicking the + icon. As soon as one of the specified conditions applies, a value will be shown (i.e. conditions are combined using "OR"). DataMiner 10.2.9/10.3.0 switches to more efficient server-side filtering, which greatly improves the filter performance but does not allow multiple conditions in the same filter.
+
+1. Click *Apply filter*.
+
+   ![Column filtering](~/user-guide/images/Column_Filtering.gif)<br>*Table component in DataMiner 10.5.6*
+
+> [!NOTE]
+> If you apply several column filters or apply both the general filter and one or more column filters, values will only be shown if they match all filters (i.e. filters are combined using "AND").
+
+## Filter based on text string
+
+From DataMiner 10.3.0 [CU20]/10.4.0 [CU8]/10.4.11 onwards<!--RN 40793-->, you can **filter the table by passing it a text string**.
```

Comment:
Important to note here that this is leveraging the client-search capability. Meaning it has the same benefits and drawbacks. Additionally, it's also possible to hook up feeds to server filtering, but this requires hooking up the filter value in the GQI query directly.